tensorboard/webapp/metrics/views/card_renderer/card_view_test.ts

```diff
@@ -93,7 +91,7 @@ describe('card view test', () => {
     fixture.componentInstance.pluginType = PluginType.SCALARS;
     fixture.detectChanges();
 
-    expect(fixture.debugElement.classes['full-width']).toBe(false);
+    expect(fixture.debugElement.classes['full-width']).not.toBeTruthy();
```

Comment:
After CardViewContainer's `@HostBinding('class.full-width') showFullWidth: boolean = false` finishes change detection, we originally expected `debugElement.classes['full-width'] === false` internally.

In OSS, `debugElement.classes['full-width'] === undefined`, so we need to be a little more relaxed.
It's unclear what the exact discrepancy is, but most likely due to a different Angular version.

Reply:
After some further hunting, this breaking change from Angular 9 (used in OSS) --> Angular 10 looks like a possible reason:

"DebugElement.classes returns undefined for classes that were added and then subsequently removed (previously, classes added and later removed would have a value of false)."
https://angular.io/guide/ivy-compatibility#less-common-changes